distro/openshift-template/mt/tenant-manager-template.yaml

```diff
@@ -0,0 +1,385 @@
+          - /configs/envoy/main.yaml
+          ports:
+          - name: api-envoy
+            protocol: TCP
+            containerPort: 9001
+          - name: metrics-envoy
+            protocol: TCP
+            containerPort: 9000
+          livenessProbe:
+            httpGet:
+              path: /ready
+              port: 9000
+            initialDelaySeconds: 10
+            timeoutSeconds: 1
+            periodSeconds: 10
+            successThreshold: 1
+            failureThreshold: 10
+          readinessProbe:
+            httpGet:
+              path: /ready
+              port: 9000
+            initialDelaySeconds: 10
+            timeoutSeconds: 1
+            periodSeconds: 10
+            successThreshold: 1
+            failureThreshold: 10
+          resources:
+            requests:
+              memory: 256Mi
+              cpu: 250m
+            limits:
+              memory: 512Mi
+              cpu: 500m
+
+- kind: PodDisruptionBudget
+  apiVersion: policy/v1
+  metadata:
+    name: tenant-manager-pdb
+  spec:
+    maxUnavailabe: "50%"
+    selector:
+      matchLabels:
+        app: tenant-manager
+        template: tenant-manager-registry
+
+parameters:
+
+- name: TENANT_MANAGER_LOG_LEVEL
+  value: DEBUG
+
+- name: TENANT_MANAGER_QUARKUS_LOG_LEVEL
+  value: INFO
+
+- name: ENABLE_ACCESS_LOG
+  value: "false"
+
+- name: SERVICE_ACCOUNT_NAME
+  displayName: Service Account to use for the deployment
+  required: true
+
+- name: REPLICAS
+  description: Number of replicas for apicurio-registry deployment.
+  value: "3"
+- name: TENANT_MANAGER_REPLICAS
+  description: Number of replicas for tenant-manager deployment.
+  value: "3"
+
+- name: TENANT_MANAGER_IMAGE_REPOSITORY
+  displayName: Tenant Manager Image Repository
+  required: true
+
+- name: IMAGE_TAG
+  displayName: Image tag for Apicurio Registry and Tenant Manager
+  value: latest
+
+- name: TENANT_MANAGER_MEMORY_REQUEST
+  description: Tenant Manager Memory request.
+  value: "600Mi"
+
+- name: TENANT_MANAGER_MEMORY_LIMIT
+  description: Tenant Manager  Memory limit.
+  value: "1000Mi"
+
+- name: TENANT_MANAGER_CPU_REQUEST
+  description: Tenant Manager  CPU request.
+  value: "200m"
+
+- name: TENANT_MANAGER_CPU_LIMIT
+  description: Tenant Manager CPU limit.
+  value: "500m"
+
+- name: DATABASE_SECRET_NAME
+  description: Secret name with database connection details
+  required: true
+- name: DATABASE_SSL_CA_SECRET_NAME
+  description: Secret name with database SSL CA
+  required: true
+
+- name: AUTH_ENABLED
+  description: flag to enable/disable authentication
+  value: "false"
+- name: KEYCLOAK_URL
+  description: Keycloak base url
+  value: "http://localhost:8090"
+- name: KEYCLOAK_REALM
+  description: Keycloak realm
+  value: "apicurio-registry"
+- name: KEYCLOAK_API_CLIENT_ID
+  description: client id for Apicurio Registry API
+  value: registry-api
+
+- name: TENANT_MANAGER_URL
+  value: http://tenant-manager:8585
+- name: TENANT_MANAGER_CA_BUNDLE
```

Comment:
The other way around from the other PR. These env vars, related to the certificates and the auth client can be removed from here.

Reply:
removed